concordia/views.py

```diff
@@ -1,20 +1,34 @@
 import csv
 import os
 import sys
+import shutil
+from django.http import HttpResponse
 from logging import getLogger
-
 import requests
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.auth.models import User
+from django.views.generic import TemplateView
+from registration.backends.simple.views import RegistrationView
 from django.core.paginator import Paginator
 from django.db.models import Count, Q, Sum
 from django.http import HttpResponse
 from django.shortcuts import get_object_or_404, redirect, render
 from django.urls import reverse
-from django.views.generic import TemplateView
-from registration.backends.simple.views import RegistrationView
+from django.contrib.auth.models import User
+from django.shortcuts import get_object_or_404
+from django.db import transaction
+from django.db.models import Q
+from django.db.models import Count, Sum
+
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+BASE_DIR = os.path.dirname(PROJECT_DIR)
+
+sys.path.append(BASE_DIR)
+
+sys.path.append(os.path.join(BASE_DIR, 'config'))
+from config import Config
```

Comment:
This looks like it hasn't been reconciled with the configuration simplification @rstorey recently made. In particular I don't see how this code could have been tested since there's no config module in the project any more.

It's also not used so it should a) be removed and b) `flake8` should always be used to clear out undeclared or unused variables before submitting code:

```
$ pipenv run flake8 concordia/views.py
concordia/views.py:1:1: F401 'csv' imported but unused
concordia/views.py:4:1: F401 'shutil' imported but unused
concordia/views.py:16:1: F811 redefinition of unused 'HttpResponse' from line 5
concordia/views.py:16:1: F401 'django.http.HttpResponse' imported but unused
concordia/views.py:19:1: F811 redefinition of unused 'User' from line 11
concordia/views.py:20:1: F811 redefinition of unused 'get_object_or_404' from line 17
concordia/views.py:22:1: F811 redefinition of unused 'Q' from line 15
concordia/views.py:23:1: F811 redefinition of unused 'Count' from line 15
concordia/views.py:23:1: F811 redefinition of unused 'Sum' from line 15
concordia/views.py:23:1: F401 'django.db.models.Sum' imported but unused
concordia/views.py:31:1: E402 module level import not at top of file
concordia/views.py:31:1: F401 'config.Config' imported but unused
concordia/views.py:33:1: E402 module level import not at top of file
concordia/views.py:34:1: E402 module level import not at top of file
concordia/views.py:36:1: E402 module level import not at top of file
concordia/views.py:36:1: F401 'importer.importer.tasks.check_completeness' imported but unused
concordia/views.py:36:1: F401 'importer.importer.tasks.download_async_collection' imported but unused
concordia/views.py:165:9: F841 local variable 'context' is assigned to but never used
concordia/views.py:221:9: F841 local variable 'context' is assigned to but never used
concordia/views.py:229:20: E225 missing whitespace around operator
```